jcl/src/java.management/share/classes/java/lang/management/ThreadInfo.java

```diff
@@ -793,7 +793,6 @@ public String toString() {
 					++stackDepth;
 				}
 			} else {
-				result.append(" null"); //$NON-NLS-1$
 				result.append(ls);
```

Comment:
You can move the `append(ls)` above the if case, then the else case isn't needed.